src/entity/Entity.js

```diff
@@ -18,17 +18,25 @@ const variableToObject = (obj) => {
 };
 
 /**
- * This function checks if the type of param is among the list : expectedTypes
- * @param {*} expectedTypes the list with the expected types
- * @param {*} param the parameter which has to be checked
- * @returns a boolean - true if typeof param is among the list, false if not
+ * This function is usefull to know if the current type is a primive type
+ * ['string','number','boolean',...]
+ * we use : typeof (type) === 'string' because this is true only when the type is a primitive one
```

Comment:
This line isn't useful - it can be remove. It is implicit since it is the only line in the function